### ISSUE TITLE
Inference Latency Discrepancy

### ISSUE THREAD

**Issue by HuuuangJW** [ShuangLI59/unified_video_action#30]
Hi, thank you for the impressive work on UVA. I am currently reproducing the results, but I’ve observed a significant discrepancy 2.2s(approx. 8-10x) between my measured inference latency and the benchmarks reported in Table I of the paper.

I would like to clarify the profiling methodology used in the paper to ensure my setup is not hitting a bottleneck.

To measure the latency, I modified `pusht_image_runner.py` by adding `torch.cuda.synchronize()` and `time.time()` around the `policy.predict_action()`. I also excluded the first 10 iterations to account for CUDA warm-up.
```
t_total = []
while not done:
    # create obs dict
    np_obs_dict = dict(obs)

    if self.past_action:
        if len(past_action_list) > 1:  ## get 16 actions
            np_obs_dict["past_action"] = np.concatenate(
                past_action_list, axis=1
            )

    # device transfer
    obs_dict = dict_apply(
        np_obs_dict, lambda x: torch.from_numpy(x).to(device=device)
    )
    torch.cuda.synchronize()
    t = time.time()
    # run policy
    with torch.no_grad():
        action_dict = policy.predict_action(obs_dict, **kwargs)
    torch.cuda.synchronize()
    t_total.append(time.time() - t)
    # device_transfer
    np_action_dict = dict_apply(
        action_dict, lambda x: x.detach().to("cpu").numpy()
    )

    action = np_action_dict["action"]  # (56, 8, 2)

    # step env
    obs, reward, done, info = env.step(action)
    # obs['image'] (56, 2, 3, 96, 96)
    done = np.all(done)

    # past_action = action
    past_action_list.append(action)
    if len(past_action_list) > 2:
        past_action_list.pop(0)

    # update pbar
    pbar.update(action.shape[1])
print(f"用时{np.mean(t_total[10:]) :.4f}")
```
In Table I, the paper states the comparison is based on the inference time for an action chunk. However, given the ~10x difference, I am wondering: Is it possible that the reported values in Table I represent the "average time per individual action" (Total Chunk Latency / Chunk Size) rather than the wall-clock time for a single forward pass (predict_action)?

Could you provide the profiling script used for the benchmarks in Table I? I would be very appreciative.

Thank you for your guidance!
